src/base/projects/View.svelte

```diff
@@ -169,11 +169,13 @@
         branches={result.branches}
         profile={result.profile}
         on:routeParamsChange={updateRouteParams} />
```

Comment:
Okay I tried some ways to remove the `routeParamsChange` event from the `Header` but I think for now it works and it does what it supposed to do quite nicely.
Maybe in the future we could remove/improve

Reply:
Yeah it makes a bit more sense there because it's triggered by a click, vs. just `onMount`